core/domain/change_domain.py

```diff
@@ -125,7 +127,7 @@ class BaseChange:
         'user_id_attribute_names': []
     }]
 
-    def __init__(self, change_dict):
+    def __init__(self, change_dict: Dict[str, str]) -> None:
```

Comment:
Function above this one is not typed.

Reply:
Also this file is not assigned to you.

Reply:
In `config_domain.py` class `ConfigPropertyChange` is extended from (`change_domain.py -> Basechange`)
https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/domain/config_domain.py#L155

so, every call to this class in `config_domain_test.py` requires proper annotation, that's why i added annotations to this class(or specific class).

yeah!, this file is not assigned to me. so, that's the reason i added only changes that are required in `config_domain.py`

Reply:
Ok, seems fine.